### ISSUE TITLE
Size of buttons

### ISSUE THREAD

**Issue by SewerSnake** [KoolP/Coffee-Break#2]
The buttons don't rescale when more characters are added. Does not quite fit when I used the text "Classrooms" instead of "Clasrooms".

**Comment by KoolP** [comment]
Yes, and when translating to Finnish the words will become even bigger:) To be, standard size in buttons so all are same to one another, or not to be scaling button so all text will fit? Thats the question my friend.

**Comment by SewerSnake** [comment]
Ha-ha! :-) That's quite a conundrum! We'll have to look into that, my friend!
